src/server/protocol.d.ts

```diff
@@ -126,6 +126,14 @@ declare module ts.server.protocol {
     }
 
     /**
+      * Go to type request; value of command field is
+      * "type". Return response giving the file locations that

```

Comment:
That sounds vague enough that I could see it becoming a problem, let's make it "gototype" or something